### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by ishaan701**

### CODE REVIEW

File: components/header.ftd [hunk 1]
```diff
@@ -8,6 +8,7 @@ module site-data: site-data
 module user-data: user-data
 optional string product:
 ds.gap margin: $ds.spaces.vertical-gap.zero
+optional boolean $sidebar: false
```
Comment (Arpita-Jaiswal) [inline components/header.ftd:11]: Are you using this anywhere? @ishaan701

File: components/slides/page-with-sidebar.ftd [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+inset: $ds.spaces.inset-square.medium
+;;border-width.px: 1
+;;border-color: $inherited.colors.border
+spacing: $ds.spaces.horizontal-gap.small
+
+-- ds.copy-regular: 
+text: Team:
+align: inline
+
+-- ds.row::
```
Comment (Arpita-Jaiswal) [inline components/slides/page-with-sidebar.ftd:68]:
```suggestion
-- ds.row:
```

File: components/header.ftd (hunk 1 above)
Comment (ishaan701) [reply components/header.ftd:11]: Its usage is present in other files

File: components/slides/page-with-sidebar.ftd [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+-- end: ds.column
+
+-- ds.column:
+width.fixed.px if { page.sidebar }: 1140
+width: fill-container
+
+-- ui:
+for: $ui in $page.uis
+
+-- end: ds.column
```
Comment (Arpita-Jaiswal) [inline components/slides/page-with-sidebar.ftd:178]:
```suggestion
-- ds.column:
width.fixed.px if { page.sidebar }: 1140
width: fill-container
content: $page.uis

-- end: ds.column
```

File: components/header.ftd (hunk 1 above)
Comment (Arpita-Jaiswal) [reply components/header.ftd:11]: Is it used in `header` component itself? @ishaan701

Comment (ishaan701) [reply components/header.ftd:11]: yes

**Pull request merged**